climada/engine/uncertainty_quantification/test/test_uncertainty.py

```diff
@@ -0,0 +1,801 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Test uncertainty module.
+"""
+
+import unittest
+import copy
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import scipy as sp
+
+from pathos.pools import ProcessPool as Pool
+
+from climada.entity import ImpactFunc, ImpactFuncSet
+from climada.entity.entity_def import Entity
+from climada.entity import Exposures
+from climada.hazard import Hazard
+from climada.engine.uncertainty_quantification import InputVar, CalcImpact, UncOutput, CalcCostBenefit
+
+from climada.util.constants import EXP_DEMO_H5, HAZ_DEMO_H5, ENT_DEMO_TODAY, ENT_DEMO_FUTURE
+from climada.util.constants import  TEST_UNC_OUTPUT_IMPACT, TEST_UNC_OUTPUT_COSTBEN
+from climada.util.api_client import Client
+
+apiclient = Client()
+ds = apiclient.get_dataset(name=TEST_UNC_OUTPUT_IMPACT)
+[test_unc_output_impact] = apiclient.download_dataset(ds)
```

Comment:
```suggestion
_target_dir, [test_unc_output_impact] = apiclient.download_dataset(ds)
```

Reply:
I get `ValueError: not enough values to unpack (expected 2, got 1)`

Reply:
Solved. This was due to changes in develop. After merging develop into this branch, the suggested writing works.